cla_public/apps/base/__init__.py

```diff
@@ -1,5 +1,12 @@
 "Base app"
 from flask import Blueprint
-
+from flask import request
+from cla_public.libs.utils import change_locale
```

Comment:
💭 Could we move the locale-specific functions into a `cla_public.libs.locale` namespace instead?